katas/7kyu/two-to-one.js

```diff
@@ -0,0 +1,8 @@
+function longest(s1, s2) {
+    const s1Set = new Set(s1.split(''));
+    const s2Set = new Set(s2.split(''));
+    const uniqSet = new Set([...s1Set, ...s2Set]);
```

Comment:
you've already avoided duplicates in `s1Set` and `s2Set`.
there's no sense to use the `new Set` again